### DISCUSSION TITLE
Get entities near player

### DISCUSSION THREAD

**Discussion by EmkoCZ** [Minestom/Minestom#1186] [Help] [answered]
Hello, I'm trying to get entities near the player, the only solution that comes up to my mind is to loop through the chunks around the player and get entities within these chunks. Is there any better way to get a list or an array of entities near the player?

**Comment by EmkoCZ** [comment, marked as answer]
Found the EntityFinder class

**Comment by RinesThaix** [comment]
You could also use
```java
instance.getEntityTracker().nearbyEntities(position, range, EntityTracker.Target.ENTITIES, entity -> {...});
```
